samples/langchain_quick_start.ipynb

```diff
@@ -20,14 +20,903 @@
+    "loader = MemorystoreDocumentLoader(\n",
+    "    client=client, key_prefix=\"netflix:\", content_fields=content_fields\n",
+    ")"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    " Run the `collect_async_items` function to pull documents from your Redis instance. Take a look at the first 5 documents below. Awesome, you've successfully used Cloud Memorystore for Redis as a document loader!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "iterator = loader.lazy_load()\n",
+    "for i in range(5):\n",
+    "    print(next(iterator))"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Use case 2: Cloud Memorystore for Redis as a Vector Store"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now, you'll learn how to put all of the documents you just loaded into a vector store backed by Cloud Memorystore for Redis. This will allow you to use vector search to answer user questions!"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To create a vector store, you'll start by initializing it using the `init_index` function. Include all the fields you want for metadata, and specify a vector size (768) that matches the output of your embedding model (Vertex AI's textembedding-gecko in this case)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import re\n",
+    "from langchain_google_memorystore_redis import (\n",
+    "    DistanceStrategy,\n",
+    "    HNSWConfig,\n",
+    "    RedisVectorStore,\n",
+    ")\n",
+    "\n",
+    "index_config = HNSWConfig(\n",
+    "    name=\"netflix_vs:\", distance_strategy=DistanceStrategy.COSINE, vector_size=768\n",
+    ")\n",
+    "\n",
+    "try:\n",
+    "    RedisVectorStore.init_index(client=client, index_config=index_config)\n",
+    "except redis.exceptions.ResponseError as e:\n",
+    "    if re.match(r\".*already exists\", str(e)):\n",
+    "        print(\"Index already exists, skipping creation.\")\n",
+    "    else:\n",
+    "        raise"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Initialize your embedding service. Here, you're using version 003 of Vertex AI's textembedding-gecko model. Specifying the model version is important for reproducibility and managing updates."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from langchain_google_vertexai import VertexAIEmbeddings\n",
+    "\n",
+    "embeddings_service = VertexAIEmbeddings(\n",
+    "    model_name=\"textembedding-gecko@latest\", project=project_id\n",
```

Comment:
```suggestion
    "    model_name=\"textembedding-gecko@003\", project=project_id\n",
```

Reply:
Note that for each insertion, the embedding service will be called to compute the embeddings to store in the vector table. Pricing details can be found [here](https://cloud.google.com/vertex-ai/pricing).